src/Sylius/Behat/Context/Transform/CustomerContext.php

```diff
@@ -43,6 +43,7 @@ public function __construct(CustomerRepositoryInterface $customerRepository, Fac
 
     /**
      * @Transform :customer
+     * @Transform /^made by "[^"]+"$/
```

Comment:
This should only `getCustomerByEmail` and never create a customer. Add second method to this context.